js/web/components/Balance.js

```diff
@@ -0,0 +1,76 @@
+import React, { useContext, useEffect, useState } from 'react'
+import Nina from '@nina-protocol/nina-internal-sdk/esm/Nina'
+import { Box, Typography } from '@mui/material'
+import { styled } from '@mui/system'
+
+const Balance = ({ profilePublishedReleases }) => {
+  const { ninaClient, solBalance, usdcBalance } = useContext(Nina.Context)
+  const [royaltySumForArtist, setRoyaltySumForArtist] = useState(0)
```

Comment:
we should use the word `revenue` as we are trying to get away from the `royalty` namespace